src/common/exponent/exponentHelper.ts

```diff
@@ -0,0 +1,384 @@
+     */
+    public configureExponentEnvironment(): Q.Promise<void> {
+        Log.logMessage("Making sure your project uses the correct dependencies for exponent. This may take a while...");
+        return this.changeReactNativeToExponent()
+            .then(() => {
+                Log.logMessage("Dependencies are correct. Making sure you have any necesary configuration file.");
+                return this.createExpJson();
+            }).then(() => {
+                Log.logMessage("Project setup is correct. Generating entrypoint code.");
+                return this.createExponentEntrypoint()
+                    .then(() =>
+                        this.createIndex());
+            });
+    }
+
+    /**
+     * Change dependencies to point to original react-native repo
+     */
+    public configureReactNativeEnvironment(): Q.Promise<void> {
+        Log.logMessage("Checking react native is correctly setup. This may take a while...");
+        return this.changeExponentToReactNative();
+    }
+
+    /**
+     * Changes npm dependency from react native to exponent's fork
+     */
+    private changeReactNativeToExponent(): Q.Promise<void> {
+        return this.usingReactNativeExponent()
+            .then(usingExponent => {
+                if (usingExponent) {
+                    return Q.resolve<void>(void 0);
+                }
+                return this.exponentSdk
+                    .then(sdkVersion => {
+                        if (!sdkVersion) {
+                            return XDL.Versions.facebookReactNativeVersionsAsync()
+                                .then((versions) => {
+                                    return Q.reject<void>(new Error(`React Native version not supported by exponent. Major versions supported: ${versions.join(", ")}`));
+                                });
+                        }
+                        const exponentFork = `github:exponentjs/react-native#sdk-${sdkVersion}`;
+                        return this.commandExecutor.execute("npm uninstall react-native", { silent: true })
+                            .then(() =>
+                                this.commandExecutor.execute(`npm install ${exponentFork}`, { silent: true }));
+                    });
+            })
+            .then(() => {
+                this.dependencyPackage = ReactNativePackageStatus.EXPONENT_PACKAGE;
+            });
+    }
+
+    /**
+     * Changes npm dependency from exponent's fork to react native
+     */
+    private changeExponentToReactNative(): Q.Promise<void> {
+        return this.usingReactNativeExponent()
+            .then(usingExponent => {
+                if (!usingExponent) {
+                    return Q.resolve<void>(void 0);
+                }
+                return this.commandExecutor.execute("npm uninstall react-native", { silent: true })
+                    .then(() =>
+                        this.commandExecutor.execute(`npm install react-native`, { silent: true }));
```

Comment:
You use backticks here but don't interpolate anything. Are you missing something, or can we use normal quotes here?